### ISSUE TITLE
Renaming Settings to Config and display deprecation warning until next major version bump

### ISSUE THREAD

**Issue by pkuczynski** [railsconfig/config#118]
As we renamed the gem from RailsConfig to Config I think we should also rename the global variable name from Settings to Config, keeping both access points available and displaying deprecation warning until the next major version bump.

@fredwu what do you think?

**Comment by vinceve** [comment]
This sounds good to me. But could it potentially crash with a rails class? I'm not sure if there is already a Config class or not... If not, it would be beneficial to the consistency for the user.

**Comment by yb66** [comment]
Hi there,

I just had a look at this library and as a heavy Sinatra user the first thing I thought was that `Settings`  will inevitably cause problems with Sinatra's own configuration accessor, `settings`. If I'm going to use this I'd run a fork based on that alone. Imagine differentiating variables based on case! (o_º)

Regards,
iain

**Comment by pkuczynski** [comment]
I think we already have Sinatra as part of our test scope and it seems to work fine with current setup...

**Comment by yb66** [comment]
@pkuczynski Fair enough, I'm just saying that it looks like the kind of thing that if another dev on a team came to me with two vars only differing by case then I'd be worried that a bug will appear. Seems like an accident waiting to happen.

Regards,
iain
